apps/site/src/components/pages/wordpress/header.tsx

```diff
@@ -0,0 +1,150 @@
+import { Stream } from "@cloudflare/stream-react";
+import { Box, Container, Typography, useTheme } from "@mui/material";
+import Image from "next/legacy/image";
+
+import wpHelixImage from "../../../../public/assets/new-home/wp-helix.png";
+import { BpWpIcons } from "./bp-wp-icons";
+import { EarlyAccessCTA } from "./early-access-cta";
+
+export const Header = () => {
+  const theme = useTheme();
+
+  return (
+    <Box
+      sx={{
+        pb: { xs: 20, md: 28 },
+        position: "relative",
+      }}
+    >
+      <Container
+        sx={{
+          position: "relative",
+          zIndex: 3,
+          pt: { xs: 16, md: 20 },
```

Comment:
This looks a lot more than in the design - I think it needs to be roughly half

Reply:
Nice spot, changed in 50c1dee5bd0c4230c3fde823a48214ce01f16776